py/DREAM/Formulas/PlasmaParameters.py

```diff
@@ -0,0 +1,116 @@
+
+import numpy as np
+import scipy.constants
+
+
+def getCoulombLogarithm(T, n):
+    """
+    Calculates the Coulomb logarithm according to the formula given in
+    Wesson's book "Tokamaks".
+
+    :param float T: Plasma temperature (eV).
+    :param float n: Plasma density (m^-3).
+    """
+    return 14.9 - 0.5*np.log(n / 1e20) + np.log(T / 1e3)
+
+
+def getEc(T, n):
+    """
+    Calculates the Connor-Hastie critical electric field, below which no
+    runaway electrons can be generated.
+
+    :param float T: Plasma temperature (eV).
+    :param float n: Plasma density (m^-3).
+    """
+    logLambda = getCoulombLogarithm(T, n)
+
+    c = scipy.constants.c
+    e = scipy.constants.e
+    me = scipy.constants.m_e
+    eps0 = scipy.constants.epsilon_0
+
+    return (n*logLambda*e**3) / (4*np.pi*eps0**2 * me * c**2)
+
+
+def getConnorHastieCriticalField(T, n): return getEc(T, n)
+
+
+def getED(T, n):
+    """
+    Calculates the Dreicer electric field at the given plasma temperature and
+    density, giving the electric field at which all electrons are accelerated
+    to the runaway region.
+
+    :param float T: Plasma temperature (eV).
+    :param float n: Plasma density (m^-3).
+    """
+    c = scipy.constants.c
+    e = scipy.constants.e
+    me = scipy.constants.m_e
+
+    Ec = getEc(T, n)
+
+    return Ec * me * c**2 / (e*T)
+
+
+def getDreicerElectricField(T, n): return getED(T, N)
+
+
+def getTauEETh(T, n):
+    """
+    Calculates the thermal electron-electron collision frequency.
+
+    :param float T: Plasma temperature (eV).
+    :param float n: Plasma density (m^-3).
+    """
+    mc2  = scipy.constants.physical_constants['electron mass energy equivalent in MeV'][0] * 1e6
+    vth2 = 2*T/mc2
```

Comment:
could do vth = getBetaThermal(T) here instead to ensure consistent defintions

Reply:
Good point! I just added ``getBetaThermal`` today and only replaced it in the tests, but forgot about ``PlasmaParameters.py`` :P Updated now though!